content/doc/developer/plugin-development/plugin-release-tips.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: developersection
+title: Plugin Release Tips
```

Comment:
Not sure we need it as a separate page, better to just add to Release guidelines. But whatever